modules/tensor_mechanics/include/materials/HyperbolicViscoplasticityStressUpdate.h

```diff
@@ -32,10 +32,10 @@ class HyperbolicViscoplasticityStressUpdate : public RadialReturnStressUpdate
 protected:
   virtual void initQpStatefulProperties() override;
 
-  virtual void computeStressInitialize(Real effectiveTrialStress,
-                                       const RankFourTensor & elasticity_tensor) override;
-  virtual Real computeResidual(Real effectiveTrialStress, Real scalar) override;
-  virtual Real computeDerivative(Real effectiveTrialStress, Real scalar) override;
+  virtual void computeStressInitialize(Real effective_trial_stress,
+                                       const RankFourTensor & /*elasticity_tensor*/) override;
```

Comment:
I think it is more common to have the ```/*elasticity_tensor*/``` commented out in the C file rather than the header file.  (There are a few more of these in your PR too.)